.circleci/config.yml

```diff
@@ -0,0 +1,170 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/coala-bears
+
+    docker:
+      - image: circleci/jdk8:0.1.1
+
+    parallelism: 2
```

Comment:
This line is redundant. Instead of parallelism use workflows https://circleci.com/docs/2.0/workflows/.

Reply:
Thanks alphadose, will definately  look into it